lib/config/cli.js

```diff
@@ -49,6 +49,9 @@ function getConfig(argv) {
       '    $ renovate --labels=renovate,dependency --ignore-unstable=false --log-level verbose singapore/lint-condo'
     );
     console.log('    $ renovate singapore/lint-condo singapore/package-test');
+    console.log(
+      '    $ renovate {repo} --token {token} --platform vsts --endpoint https://{instance}.visualstudio.com/DefaultCollection'
```

Comment:
The others here are *Examples* while this is kind of like a template. We should make them consistent

Reply:
I agree, just the problem is that there is no public endpoint, so I cannot provide an example.
You could put something like: `console.log('  Examples/templates:');` at the begining?